lib/middleware/proxy.js

```diff
@@ -22,23 +27,28 @@ module.exports = function(options) {
     return function(req, res, next) {
         if (req.url.indexOf('/__api__/proxy/') === 0 || req.url.indexOf('/proxy/') === 0) {
 
-            var url = decodeURIComponent(req.url.replace('/proxy/', '')),
-                targetHost = url.substring(0, url.indexOf('/', 8));
-
-            //Re-Build the location header, so browsers can follow them
+            var url = decodeURIComponent(req.url.replace('/proxy/', ''));
+			var targetHost = url.substring(0, url.indexOf('/', 8));
+			var path = url.substring(url.indexOf('/', 8), url.length);
```

Comment:
Please retain the since `var` pattern used for variable declaration:

```
var url = decodeURIComponent(req.url.replace('/proxy/', ''),
    targetHost = url.substring(0, url.indexOf('/', 8),
    path = url.substring(url.indexOf('/', 8), url.length);
```

Additionally, I don't think we should use the variable name `path` which is often reserved for `require('path');`. Instead, I'd suggest `proxyReqPath` since that is the value that it maps to.